NMDSdevopsServiceAdm/DataEngineering
# Added additional columns to ASC-WDS column selection list in clean_as…

# Description
Added master update, master update of the org and creation date to the list of columns to import in clean_ascwds_workplace_data and merge_coverage_data.

This will also affect the reconciliation job. The columns will appear in that output.


# How to test
No unit tests.
Can see new columns using Athena.

Athena clean ascwds workplace data reconciliation output:
https://eu-west-2.console.aws.amazon.com/athena/home?region=eu-west-2#/query-editor/history/cde3767c-3c99-4e2f-bbea-c8a924617f10

Athena merge coverage output:
https://eu-west-2.console.aws.amazon.com/athena/home?region=eu-west-2#/query-editor/history/bd0835c4-7ec2-41ad-b62f-c3807c4c5510

Link to successful clean_ascwds_workplace_data in Glue:
https://eu-west-2.console.aws.amazon.com/gluestudio/home?region=eu-west-2#/editor/job/import-additional-ascwds-data-clean_ascwds_workplace_data_job/runs

Link to successful merge_coverage_data in Glue:
https://eu-west-2.console.aws.amazon.com/gluestudio/home?region=eu-west-2#/editor/job/import-additional-ascwds-data-merge_coverage_data_job/runs


# Developer checklist
